README.md

```diff
@@ -280,6 +283,11 @@ We will now be able to see on the dashboard that our service is of type `NodePor
 kubectl get service sample-app-service --output='jsonpath="{.spec.ports[0].nodePort}"'
 ```
 
+You also need to forward the port for the service. This differs slightly from forwarding a pod, as you can see below.
```

Comment:
This is actually very useful, didnt know you could port forward a service 🙂

Reply:
Yep! That's what I assumed was meant in the section heading xD Took me a while to find the exact syntax, but it worked well.